loader_hub/file/audio/base.py

```diff
@@ -17,16 +17,32 @@ class AudioTranscriber(BaseReader):
 
     """
 
-    def __init__(self, *args: Any, model_version: str = "base", **kwargs: Any) -> None:
+    def __init__(self, *args: Any, model_version: str = "base", 
+                model_type: str = "openai_whisper", diarization_max_speakers: str = None,
+                language: str = "English", language_behaviour: str = "automatic multiple languages",
+                target_translation_language: str = 'english', gladia_api_key: str = None, 
+                transcription_hint: str = None, **kwargs: Any) -> None:
         """Init params."""
         super().__init__(*args, **kwargs)
-        self._model_version = model_version
 
-        import whisper
+        self.parser_config = {}
+        self.model_type = model_type
+
+        if self.model_type == "openai_gladia_whsiper":
+            self.parser_config['gladia_api_key'] = gladia_api_key
+            self.parser_config['diarization_max_speakers'] = diarization_max_speakers
+            self.parser_config['language'] = language
+            self.parser_config['language_behaviour'] = language_behaviour
+            self.parser_config['target_translation_language'] = target_translation_language
+            self.parser_config['transcription_hint'] = transcription_hint
+        else:
```

Comment:
let's do `elif` model_version == "gladia", and then explicitly raise ValueError in the `else` block if the model_version is invalid

Reply:
sure. will update.